### ISSUE TITLE
Cannot Read Property Tap Of Undefined

### ISSUE THREAD

**Issue by Quadriphobs1** [pedronauck/docz#498]
## Bug Report

**Describe the bug**

I am trying out the package with a nextjs project, this is a project which has express with next js with it. It was an ongoing project which has been working for long, and we decided to try out the docz to document the components.

**To Reproduce**

1. Install docz package
2. Run `yarn docz:dev` which is `=== docz dev`
3. TypeError: Cannot read property 'tap' of undefined during compilation

```shell
compilation.hooks.addEntry.tap("ProgressPlugin", entryAdd);
				                           ^

TypeError: Cannot read property 'tap' of undefined
```

**Expected behavior**
Since the build command runs successfully, it should also run as expected.

*A clear and concise description of what you expected to happen..*
Start the doc server locally

**Environment**

* OS: [e.g. OSX 10.13.4]
* Node/npm version: [e.g. Node 10/npm 6]

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by chhuang** [comment]
I've solved this by installing the latest canary `next@7.0.2-canary.45`.

**Comment by pedronauck** [comment]
Can you try the this canary solution @Quadriphobs1 ?

**Comment by Quadriphobs1** [comment]
👍 It works thanks, However with my old configuration it doesnt work with that, I am having `Can't reexport the named export 'isValidJSValue' from non EcmaScript module (only default export is available)` from `graphql-js` due to `webpack` configuration mismatch. Hopefully I would resolve that

**Comment by Quadriphobs1** [comment]
Finally fixed it by updating `react-apollo@2.1.0-beta.3`

**Comment by eden-lane** [comment]
I've just installed docz and I'm getting the same error.
